Tools/WebKitTestRunner/cocoa/TestControllerCocoa.mm

```diff
@@ -512,26 +512,6 @@ void initializeWebViewConfiguration(const char* libraryPath, WKStringRef injecte
     }];
 }
 
-void TestController::loadedSubresourceDomains()
-{
-    auto* parentView = mainWebView();
-    if (!parentView)
-        return;
-    
-    [[globalWebViewConfiguration() websiteDataStore] _loadedSubresourceDomainsFor:parentView->platformView() completionHandler:^(NSArray<NSString *> *domains) {
-        m_currentInvocation->didReceiveLoadedSubresourceDomains(makeVector<String>(domains));
-    }];
-}
-
-void TestController::clearLoadedSubresourceDomains()
```

Comment:
I don't understand why it's okay to remove these.

Reply:
we're using the platform-independent versions now